### PR TITLE
Update heimer_it.ts

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

**Comment by juzzlin** [comment]
Merged, thanks!

**Comment by juzzlin** [comment]
I found one error that has obviously been there also before: In the node context menu there's same Italian text twice.

File: src/translations/heimer_it.ts [hunk 1]
```diff
@@ -102,186 +102,186 @@
...
         <location filename="../main_context_menu.cpp" line="68"/>
         <source>Set node color</source>
-        <translation type="unfinished">Imposta il colore del nodo</translation>
+        <translation>Imposta il colore del nodo</translation>
     </message>
     <message>
         <location filename="../main_context_menu.cpp" line="79"/>
         <source>Set text color</source>
-        <translation type="unfinished">Imposta il colore del nodo</translation>
+        <translation>Imposta il colore del nodo</translation>
```
Comment (juzzlin) [inline src/translations/heimer_it.ts]: This is apparently wrong translation for "Set text color" ?

**Comment by juzzlin** [comment]
Unfortunately I already merged this so that could be fixed in another PR..

**Comment by albanobattistella** [comment]
Sorry...I fix it!

thanks for your great work!


Albano

________________________________
Da: Jussi Lind <notifications@github.com>
Inviato: venerdì 31 gennaio 2020 19:33
A: juzzlin/Heimer <Heimer@noreply.github.com>
Cc: albanobattistella <albano_battistella@hotmail.com>; Author <author@noreply.github.com>
Oggetto: Re: [juzzlin/Heimer] Update heimer_it.ts (#80)


Unfortunately I already merged this so that could be fixed in another PR..

**Pull request merged**